### PR TITLE
[Keyboard] Add mkh studio bully

### PR DESCRIPTION
**Pull request by zhol0777**
## Description

This adds QMK support for a keyboard called "bully."

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mkh_studio/bully/config.h [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+#pragma once
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Use 1000hz polling */
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (fauxpark) [inline keyboards/mkh_studio/bully/config.h:11]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mkh_studio/bully/bully.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	K0A, K0B, K0C, K0D, K0E, K0F, K0G, K0H, K0I, K0J, K0K, K0L, K3L, \
+    K1A, K1B, K1C, K1D, K1E, K1F, K1G, K1H, K1I, K1J, K1K, K1L,      \
+    K2A, K2B, K2C, K2D, K2E, K2F, K2G, K2H, K2I, K2J, K2K, K2L,      \
+    K3A, K3B, K3C, K3D,      K3F, K3G,      K3I, K3J, K3K            \
+) { \
+	{ K0A, K0B, K0C, K0D, K0E, K0F, K0G, K0H, K0I, K0J, K0K, K0L }, \
+	{ K1A, K1B, K1C, K1D, K1E, K1F, K1G, K1H, K1I, K1J, K1K, K1L }, \
+	{ K2A, K2B, K2C, K2D, K2E, K2F, K2G, K2H, K2I, K2J, K2K, K2L }, \
+	{ K3A, K3B, K3C, K3D, XXX, K3F, K3G, XXX, K3I, K3J, K3K, K3L } \
+}
```
Comment (waffle87) [inline keyboards/mkh_studio/bully/bully.h:19]: This file and `bully.c` should be removed, assuming the `matrix` data in info.json file is correct.

Comment (zhol0777) [reply keyboards/mkh_studio/bully/bully.h]: Validated, done

File: keyboards/mkh_studio/bully/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Hardware Supported: APM32F072
+* Hardware Availability: [Coffee Break Keyboards](https://cbkbd.com/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb mkh_studio/bully -km default
+
+Flashing example for this keyboard:
+
+    qmk flash -kb mkh_studio/bully -km default
```
Comment (waffle87) [inline keyboards/mkh_studio/bully/readme.md:13]:
```suggestion
    make mkh_studio/bully:default

Flashing example for this keyboard:

    make mkh_studio/bully:default:flash
```

**Review by waffle87 — APPROVED** [review]

File: keyboards/mkh_studio/bully/config.h [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```
Comment (zvecr) [inline keyboards/mkh_studio/bully/config.h:1]:
```suggestion
// Copyright 2022 zhol
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by zvecr — APPROVED** [review]

**Pull request merged**